### DISCUSSION TITLE
Mods dont seem to load

### DISCUSSION THREAD

**Discussion by SammyG43** [X2CommunityCore/xcom2-launcher#263] [Help] [answered]
everything seems to show up fine in the launcher, I select to enable mods but when I hit "run wotc" and it launches. all is fine bu it loads vanilla wotc without mods.

It is likely a simple mistake on my part but can anyone advise how to fix this?

**Comment by RevZero** [comment, marked as answer]
Make sure, that you do NOT have the `-regenerateinis` argument enabled.

In the Settings dialog "Active arguments", it should look like this:
![image](https://user-images.githubusercontent.com/936992/109794418-23ff7100-7c16-11eb-8b7f-564c9522f32c.png)
